src/pages/AddJob.jsx

```diff
@@ -49,134 +49,243 @@ export default function AddJob() {
+                    placeholder="Select a date"
+                    {...register("dateJobStart", { setValueAs: (v) => new Date(v) })}
+                  />
+                  <label
+                    htmlFor="dateJobStart"
+                    className="pointer-events-none absolute top-0 left-3 mb-0 max-w-[90%] origin-[0_0] truncate pt-[0.37rem] leading-[2.15] text-neutral-500 transition-all duration-200 ease-out peer-focus:-translate-y-[1.15rem] peer-focus:scale-[0.8] peer-focus:text-primary peer-data-[te-input-state-active]:-translate-y-[1.15rem] peer-data-[te-input-state-active]:scale-[0.8] motion-reduce:transition-none dark:text-neutral-200 dark:peer-focus:text-neutral-200"
+                  ></label>
+                </div>
               </div>
             </div>
-            <Controller
-              name="dateInput"
-              control={control}
-              render={({ field }) => (
-                <DatePicker
-                  className="input w-full max-w-xs input-bordered mb-4"
-                  // className="bg-gray-50 border border-gray-300 text-gray-900 text-sm rounded-lg focus:ring-blue-500 focus:border-blue-500 block w-full pl-10 p-2.5 dark:bg-gray-700 dark:border-gray-600 dark:placeholder-gray-400 dark:text-white dark:focus:ring-blue-500 dark:focus:border-blue-500"
-                  placeholderText="Select Start Date"
-                  onChange={(date) => field.onChange(date)}
-                  selected={field.value}
-                />
-              )}
-            />
+          </div>
+
+          <div className="dropdown dropdown-right flex flex-row mb-4 items-center w-1/2">
+            <label className="label flex basis-44 ml-4" htmlFor="jobType">
+              Job Type
+            </label>
+            <select
+              className="input input-bordered w-full"
+              {...register("jobType", { required: true })}
+            >
+              {Object.values(SHIFT_TYPE).map((type, index) => {
+                return (
+                  <option value={type} key={index}>
+                    {type}
+                  </option>
+                );
+              })}
+            </select>
           </div>
         </div>
-        <div className="form-control">
-          <label className="label" htmlFor="wage">
-            Wages
+
+        <div className="flex flex-row items-center w-full mb-4">
+          <label className="label flex basis-44" htmlFor="shift">
+            Shift Detail
           </label>
-          <div style={{ display: "inline-flex" }}>
-            <label className="input-group input-md">
-              <span>Min Wage</span>
+          <textarea
+            type="text"
+            rows="3"
+            className="textarea textarea-bordered w-full"
+            placeholder="Please enter shift details in the following format:
+            Monday-Tuesday 8:30 AM to 1 PM
+            Thursday - Friday 1 PM to 5 PM"
+            {...register("shift")}
```

Comment:
Prompt helps users to understand what is expected in which fashion. This will also help in filtering or searching keywords as the format will help the user to enter minimum words.